### DISCUSSION TITLE
[solved] Your account is currently locked from purchases. Please update your payment information.

### DISCUSSION THREAD

**Discussion by ShinChven** [community/community#19123] [Copilot Conversations] [answered]
I'm seeing an error saying `Your account is currently locked from purchases. Please update your payment information.` after I submit the GitHub Copilot payment setup with a PayPal account.

Anyone seeing this, too?

**Comment by leoneltst** [comment]
me too. . .:c

**Comment by davecheney** [comment]
Please contact GitHub support to resolve this

**Reply by ShinChven** [reply]
I opened a ticket, no response for now.

**Comment by ShinChven** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/3351486/175186776-4569592e-03bc-4e55-9b97-df97f29753c1.png)
Problem solved by contacting GitHub support. It's actually a billing state problem.

**Comment by Kinkar2580** [comment]
Whate'sOtp
